app/views/booking_requests/step_one.html.erb

```diff
@@ -50,6 +50,9 @@
         <%= f.button('Continue', type: 'submit', class: 'button t-continue') %>
       </p>
     <% end %>
+    <p>This service is in development.</p>
+
+    <p>You may not get one of the 3 dates you’ve requested. <br>If there’s a  problem, we’ll call or email to confirm an appointment date with you.</p>
```

Comment:
@benlovell I showed it to content guys all as one paragraph but after changing it around it a few times it was thought this would be the best for it to be displayed.

@benbarnett I'll sort the extra space out